src/components/PageSections/Header/_Header.scss

```diff
@@ -1,6 +1,8 @@
 .coa-Header {
-  border-bottom: 1px solid $coa-color-purple-lighter;
+  border-bottom: 1px solid lighten( $coa-color-white, 10% );
```

Comment:
not sure why we don't use these SASS features so I'm happy to refactor this to match other color variables